README.md

```diff
@@ -45,13 +45,37 @@ components/
     index.js                    # This just exports your [Component].jsx
     [Component].jsx             # The main component file
     [Component].test.jsx        # Unit tests for the component
+    [Component].scss            # component-specific styles
   util/                         # Utility functions shared by components
+styles/
+  partials/
+    base.scss                   # Widely used styles throughout all components.
+    normalize.scss              # CSS reset/normalize
+  core.scss                     # Combines the component specific scss partials and exports to core.css
+  quartz-react.scss             # Combines the partials and every single component style, exports to quartz-react.css
+  variables.scss                # Variables used in any stylesheet, such as colors.
 demo/
   public/                       # Assets to be used by the demo page
   src/
     sections/                   # Sections of the demo page with code and examples
     ui/                         # Components specific to the demo page that are not included in quartz-react
     index.jsx                   # The demo page itself (ie. what is seen on localhost:3000)
+coverage/                       # Generated code coverage reports for component tests
+  [Component]/
+    [Component].jsx.html
+    [Component].test.jsx.html
+    index.html
+  base.css                      # code coverage css
```

Comment:
I think that you can exclude all the coverage stuff maybe besides index.html. You will never be touching any of this since it's generated, and if `nyc` which generates the coverage reports changes its implementation, we'd need to change our docs, so this could very quickly be out of date.